data-transport-layer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:10.23-buster as build
+
+RUN apt-get update \
+    && apt-get install -y bash git python build-essential libusb-1.0
+
+ARG REMOTE=https://github.com/ethereum-optimism/data-transport-layer
+ARG BRANCH=master
+ARG OUTDIR=/opt/data-transport-layer
+
+RUN git clone $REMOTE $OUTDIR \
+    && cd $OUTDIR \
+    && git checkout $BRANCH \
+    && yarn install \
+    && yarn build
+
+FROM node:10.23-buster
+RUN apt-get update \
+    && apt-get install -y bash curl jq python3 libusb-1.0 \
```

Comment:
Nit: I don't think we technically need python3 or libusb-1.0 here as well as libusb-1.0 in the above. I'm on with leaving libusb but getting rid of python3 means we can get rid of the symlink below. It was used for python simple http server in the deployer to start a server after the deployment was done, so that a `wait-for...` script could ping for the deployment to be finished

Reply:
Oh nice, I missed that one.